### ISSUE TITLE
ncc on windows is not working since version 0.19.0

### ISSUE THREAD

**Issue by GiladShoham** [zeit/ncc#439]
Since version 0.19.0, ncc on windows does nothing.
Any command running (even just `ncc` without any argument) just print an empty line.
reverting back to v0.18.5 and it's working again.

**Comment by GiladShoham** [comment]
Also found out that in v0.18.5 of ncc the following is not working:
```sh
npm init -y
npm i requirejs
touch a.js
echo "require('requirejs')" > a.js
ncc build a.js
```
Which makes my windows machine not working since I use this package and newer versions of ncc are not working on windows.

**Comment by guybedford** [comment]
Thanks so much @GiladShoham for the report. This should be fixed in 0.20.2. Please let me know if you still notice any problems though.

**Comment by rognales** [comment]
Thanks guys. 0.20.2 solved it.
